challenges/objects-arrays.js

```diff
@@ -47,23 +81,36 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 {"id":10,"first_name":"Hube","university":"Universitat Rovira I Virgili Tarragona","email":"hlethbrig9@foxnews.com"}]
 
 /* Request 1: Create a new array called universities that contains all the universities in the graduates array.  
-
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
-console.log(universities)
 
-/* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
+//THIS CODE ALSO WORKS AND DOESN'T REQUIRE ACCUMULATOR.
+// const universities = graduates.map(graduates => graduates.university);
+
+graduates.forEach((graduates) => {
+  return universities.push(`${graduates.university}`);
+})
 
+console.log(universities);
+
+/* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
 The resulting contact information should have a space between the first name and the email information like this: 
 Name email@example.com
-
 Log the result of your new array. */
 const contactInfo = [];
+
+graduates.forEach((graduates) => {
+  return contactInfo.push(`${graduates.first_name} ${graduates.email}`)
+});
+
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
-const uni = [];
+// const uni = [];
+
```

Comment:
This isn't returning the universities, this is only returning true or false. The array should contain the university.